### ISSUE TITLE
bug: Tinyllama does not load on 0.4.0-41 nightly build with error message

### ISSUE THREAD

**Issue by dan-jan** [janhq/jan#976]
**Describe the bug**
- Unable to load Tinyllama on 0.4.0-41 nightly with Windows hotfixes, even after 5 minutes
- Error message in console indicates Nitro process has exited

**To Reproduce**
0. Delete old `/jan` folder from C:/Users/<User-name>/jan
1. Install fresh copy of 0.4.0-41
2. Download Tinyllama
3. Start new Thread with Tinyllama, model fails to load, error appears in console

```
Uncaught (in promise) Error: Error invoking method `invokeExtensionFunc`: Nitro process exited. 1 
index.js:2275
```

**Screenshots**
![image](https://github.com/janhq/jan/assets/101145494/d00800b2-62e3-4ade-a20e-62260264c681)

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro
 - 11th Gen Intel i7-1160G7 @ 1.20Ghz 2.11Ghz
 - 16gb RAM
 - No GPU

![image](https://github.com/janhq/jan/assets/101145494/22d8be2c-1739-4fde-b179-bc8865ca7780)

**Additional context**

**Comment by dan-jan** [comment]
Issue resolved by running Jan `0.4.0-41` nightly on separate user on same machine. Issue likely due to orphaned Nitro process from previous QA process. Closing issue. 

Let's go ahead to cut 0.4.1 release!

**Comment by dan-jan** [comment]
Issue is tied to: https://github.com/janhq/jan/issues/972
